modules/apps/frontend-js/frontend-js-metal-web/.npmbundlerrc

```diff
@@ -21,5 +21,6 @@
 		]
 	},
 	"output": "build/node/packageRunBuild/resources",
-	"preset": "liferay-npm-bundler-preset-standard"
+	"preset": "liferay-npm-bundler-preset-standard",
+	"source": ["src/main/resources/META-INF/resources"]
```

Comment:
I believe this is `sources` instead of `source` :point_right: https://github.com/liferay/liferay-js-toolkit/wiki/.npmbundlerrc-file-reference#sources